src/ui_parts/tag_container.gd

```diff
@@ -1,6 +1,8 @@
 extends PanelContainer
 
-const safe_margin = 1 / 6.0
+# Autoscroll area on drag and drop. As a factor from edge to center.
+const safe_margin := 1.0 / 2.0
```

Comment:
Maybe i should rename the variable... its not exactly the safe margin.
Its the scroll area.

Reply:
Sure.

Are you sure about 1/2? Seems quite extreme.

Reply:
1/2 is actually 1/4 of the scroll container height.
Also keep in mind scrolling is slower near the center.

Reply:
I know, and I know, but it seems extreme nonetheless. You normally only have scrolling towards the very edges, the central part is for dropping things without repercussions.

Reply:
Tweaked it from 1/2 to 1/3, it seemed_ best to me that way. With 1/2, I had situations where I grab a tag and it immediately starts scrolling.